### ISSUE TITLE
Metadata from TCGA studies in TCIA does not include the disease

### ISSUE THREAD

**Issue by ianfore** [biocaddie/prototype_issues#284]
This is the dataset in the Genomic Data Commons
https://datamed.org/display-item.php?repository=0026&id=596a530e5152c6133df72353
It correctly indexes the disease as Head and Neck Squamous Cell Carcinoma

Images from the same study (TCGA-HNSC) are in the TCIA. This is the image dataset in TCIA for the same disease
https://datamed.org/display-item.php?repository=0016&id=5951827f5152c64c3b0ad94d
This dataset is not indexed by the disease.

**Comment by cmungall** [comment]
Same when we search using the API:

https://datamed.org/webapi/esearch?searchtype=data&query=TCGA-HNSC

the TCIA record has
```
"disease": [
{
"name": "cancer"
}
"anatomicalPart": [
{
"name": "Head-Neck"
}
```

I recommend using disease and anatomy IDs from relevant ontologies.

The NLP fields don't seem to pick this up:

```
NLP_Fields": {
"CellLine": [],
"MeshID": [
"C3273927"
],
"DiseaseID": [],
"ChemicalID": [],
"Disease": [],
"Chemical": [],
"BP": [],
"GOID": [],
"gene": [],
"GeneID": [],
"Meshterm": [
"the cancer genome atlas"
]
}
```

I can't find C3273927 in MESH - what is this ID for?

**Comment by ianfore** [comment]
That ID is in the NCI Metathesaurus as ID for the Cancer Genome Atlas
https://ncim.nci.nih.gov/ncimbrowser/ConceptReport.jspdictionary=NCI%20Metathesaurus&code=C3273927

**Comment by cmungall** [comment]
Yep - looks like it should go in a UMLS field rather than meshId
